src/core/ext/filters/client_channel/xds/xds_bootstrap.h

```diff
@@ -19,6 +19,8 @@
 
 #include <grpc/support/port_platform.h>
 
+#include <memory>
```

Comment:
What is `memory` for?

Reply:
It's for `std::unique_ptr<>`.  This should have been here before this PR anyway, so I fixed it while I was in here.